tests/integration/conftest.py

```diff
@@ -242,9 +242,16 @@ def fail_on_errors():
 
 @pytest.fixture
 def s3(pytestconfig):
+    """AWS S3 credentials for testing S3 buckets.
+
+    This will skip if the secrets are not in the environment, unless we are running on CI in
+    the getsentry team, in which case it will fail.  When running CI not as part of the
+    getsentry team you do not automatically get access to the configured secrets so skipping
+    is allowed.
+    """
     if not AWS_ACCESS_KEY_ID or not AWS_SECRET_ACCESS_KEY:
         msg = "No AWS credentials"
-        if pytestconfig.getoption("ci"):
+        if os.getenv("CI") and os.getenv("GITHUB_REPOSITORY").startswith("getsentry/"):
```

Comment:
I’m not sure this is correct.
Maybe you were inspired by https://github.com/getsentry/sentry-native/blob/dfc96e3fcbe162b84fd1dd71ca65b6387eac0ca2/.github/workflows/ci.yml#L152 ?

However that doesn’t really work as seen here: https://github.com/getsentry/sentry-native/pull/413/checks?check_run_id=1358094950 :-(

I’m not sure there is a good solution to "is this PR from a fork or internal"?

Reply:
arguably it should check for the full `getsentry/symbolicator` in the `GITHUB_REPOSITORY` env var.  But otherwise I think this should be fine?  Ok, some tests are not required for external contributors, but at least our master will break if that was bad.